cl.conf

```diff
@@ -133,13 +133,15 @@ storage:
           sudo curl -Lo $MK https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 && sudo chmod +x $MK
           sudo curl -Lo $KUBECTL https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl && sudo chmod +x $KUBECTL
 
+          cp -af $KUBECTL $USER_HOME/.bricks/bin/kubectl
```

Comment:
I don't think this will work because this kubectl will be built for linux. We should do this in the bricks_bootstrap anyway.

Reply:
Ah yes, we can just download kubectl for mac here as well and copy it over to appropriate directory on host, similar to what we do for docker binary.